rl4co/models/__init__.py

```diff
@@ -19,7 +19,7 @@
 from rl4co.models.rl.ppo.ppo import PPO
 from rl4co.models.rl.reinforce.baselines import REINFORCEBaseline, get_reinforce_baseline
 from rl4co.models.rl.reinforce.reinforce import REINFORCE
-from rl4co.models.zoo import HetGNNModel
+from rl4co.models.zoo import L2DModel
```

Comment:
Good - let's make sure the baselines have their names! L2D is a very influential paper in the NCO community